### PR TITLE
Admin page and member login page responsiveness.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
Admin page and member login page responsiveness.

### CODE REVIEW

File: src/components/common/fields/InputField.jsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ function InputField(props) {
     }
   }
   return (
-    <Col padding="0" size={12}>
+    <Box backgroundColor="unset" borderStyle="none" padding="0">
```
Comment (pepcus-pratik) [inline src/components/common/fields/InputField.jsx]: **Box** should not be used here. This line should be removed.

File: src/components/common/fields/SelectList.jsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ function SelectList(props) {
   const getSelectList = () => {
     if (enums) {
       return (
-        <Col size={12}>
+        <Box backgroundColor="unset" borderStyle="none" padding="0">
```
Comment (pepcus-pratik) [inline src/components/common/fields/SelectList.jsx]: Same here

File: src/components/routeComponents/splashPage/LoginForm.jsx [hunk 3]
```diff
@@ -61,19 +72,21 @@ const LoginForm = ({
...
+          margin="10px 15px"
           onClick={redirectToNewRegistrationPage}
         >
           {newRegistrationBtnText}
-        </Button>
-        <Button
-          margin="10px"
+        </ButtonStyled>
+        <ButtonStyled
+          width="185px"
```
Comment (pepcus-pratik) [inline src/components/routeComponents/splashPage/LoginForm.jsx]: Why do we need to hard-code the button width?

File: src/components/routeComponents/splashPage/SplashPage.jsx [hunk 4]
```diff
@@ -288,12 +302,12 @@ class SplashPage extends Component {
     return (
       <ContainerStyled width="100%">
         <BoxStyled
+          width="600px"
```
Comment (pepcus-pratik) [inline src/components/routeComponents/splashPage/SplashPage.jsx]: Why are you adding the width here when its already defined in BoxStyled CSS?

File: src/config/tenant/YJSG-Bhopal/Registration.json [hunk 5]
```diff
@@ -341,7 +341,8 @@
             "sm": 12,
             "md": 6,
             "lg": 4
-          }
+          },
+          "padding": "0 10px 0 10px"
```
Comment (pepcus-pratik) [inline src/config/tenant/YJSG-Bhopal/Registration.json]: Is this needed here?

File: src/components/common/fields/InputField.jsx (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/fields/InputField.jsx]: Done

File: src/components/common/fields/SelectList.jsx (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/common/fields/SelectList.jsx]: Done.

File: src/components/routeComponents/splashPage/LoginForm.jsx (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/splashPage/LoginForm.jsx]: Done.

File: src/components/routeComponents/splashPage/SplashPage.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/routeComponents/splashPage/SplashPage.jsx]: Done.

File: src/config/tenant/YJSG-Bhopal/Registration.json (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/config/tenant/YJSG-Bhopal/Registration.json]: Yes, this is needed.

**Pull request merged**